tests/common/database_fixture.hpp

```diff
@@ -170,6 +170,11 @@ extern uint32_t GRAPHENE_TESTING_GENESIS_TIMESTAMP;
 
 namespace graphene { namespace chain {
 
+class clearable_block : public signed_block {
```

Comment:
Please write some comments for this class.

Reply:
done